tests/dummy/app/templates/nodes/detail/draft-registrations/detail.hbs

```diff
@@ -0,0 +1,71 @@
+
+<h2> {{model.draft.registrationSchema.name}} </h2>
+<p> Initiated by: {{model.draft.initiator.fullName}}</p>
+<p> Started: {{moment-format model.draft.datetimeInitiated}} </p>
+<p> Last Updated: {{moment-format model.draft.datetimeUpdated}} </p>
+
+<hr>
+
+<h3> Edit Draft Registration </h3>
+{{#each model.draft.registrationSchema.schema.pages as |page|}}
+
+    {{#each page.questions as |question|}}
+        <p> <label> {{question.title}} </label> </p>
+        {{#if question.description}}
+            <p> {{question.description}} </p>
+        {{/if}}
+        {{#if (eq question.type "choose")}}
+            {{#unless question.description}}
+                <p> <em> Please choose </em> </p>
+            {{/unless}}
+            <form>
+                {{#each question.options as |option|}}
+                    {{#if (eq (checkDataType option) "string")}}
+                        <p> <input onchange={{action "buildForm" value="target"}} type="radio" name={{question.qid}} value={{option}} checked={{selectedHelper option (draftTextInput model.draft.registrationMetadata question)}}> {{option}} </p>
```

Comment:
For future reference... although ember [doesn't](https://guides.emberjs.com/v2.4.0/templates/writing-helpers/#toc_helper-names) *require* helper names to have a dash, this syntax is generally preferred over `camelCase` helper names.

In addition to consistency, I believe the reasoning is that it's less likely to conflict with existing JS variable names of HTML tags. (at least, that's why [web components](http://webcomponents.org/articles/how-should-i-name-my-element/) insist on hyphenated names...)